redisinsight/ui/src/utils/cliHelper.tsx

```diff
@@ -121,6 +121,18 @@ const checkUnsupportedModuleCommand = (loadedModules: RedisModuleDto[], commandL
   return commandModule
 }
 
+const getDbIndexFromSelectQuery = (query: string): number => {
+  const [command, ...args] = query.trim().split(' ')
+  if (command.toLowerCase() !== 'select') {
```

Comment:
May be better use exported constant `SelectCommand` from `cli-output` slice?

Reply:
done